lib/utils/matrix_sdk_extensions/matrix_file_extension.dart

```diff
@@ -11,9 +11,8 @@ import 'package:share_plus/share_plus.dart';
 import 'package:fluffychat/utils/platform_infos.dart';
 import 'package:fluffychat/utils/size_string.dart';
 
-const String fileName = "/storage/emulated/0/Download";
-
 extension MatrixFileExtension on MatrixFile {
+  static const String _downloadDirectoryPath = "/storage/emulated/0/Download";
```

Comment:
maybe it only works with emulator, not work with real device,

Reply:
Why don't you use path_provider package to get path ?